scripts/SMD_chip_package_rlc-etc/size_definitions/size_resistor_chip.yaml

```diff
@@ -0,0 +1,112 @@
+R_0603:
+    code_imperial: "0603"
+    code_metric: "1608"
+    body_length:
+        minimum: 1.5
+        maximum: 1.7
+    body_width:
+        minimum: 0.7
+        maximum: 0.95
+    terminal_length:
+        minimum: 0.15
+        maximum: 0.4
+    size_info: '(Body size source: IPC-SM-782 page 72, https://www.pcb-3d.com/wordpress/wp-content/uploads/ipc-sm-782a_amendment_1_and_2.pdf)'
+
+R_0805:
+    code_imperial: "0805"
+    code_metric: "2012"
+    body_length:
+        minimum: 1.85
+        maximum: 2.15
+    body_width:
+        minimum: 1.1
+        maximum: 1.4
+    terminal_length:
+        minimum: 0.15
+        maximum: 0.65
+    size_info: '(Body size source: IPC-SM-782 page 72, https://www.pcb-3d.com/wordpress/wp-content/uploads/ipc-sm-782a_amendment_1_and_2.pdf)'
+
+R_1206:
+    code_imperial: "1206"
+    code_metric: "3216"
+    body_length:
+        minimum: 3.05
+        maximum: 3.35
+    body_width:
+        minimum: 1.45
+        maximum: 1.75
+    terminal_length:
+        minimum: 0.25
+        maximum: 0.75
+    size_info: '(Body size source: IPC-SM-782 page 72, https://www.pcb-3d.com/wordpress/wp-content/uploads/ipc-sm-782a_amendment_1_and_2.pdf)'
+
+R_1210:
+    code_imperial: "1210"
+    code_metric: "3225"
+    body_length:
+        minimum: 3.05
+        maximum: 3.35
+    body_width:
+        minimum: 2.34
+        maximum: 2.64
+    terminal_length:
+        minimum: 0.25
+        maximum: 0.75
+    size_info: '(Body size source: IPC-SM-782 page 72, https://www.pcb-3d.com/wordpress/wp-content/uploads/ipc-sm-782a_amendment_1_and_2.pdf)'
+
+R_2010:
+    code_imperial: "2010"
+    code_metric: "5025"
+    body_length:
+        minimum: 4.85
+        maximum: 5.15
+    body_width:
+        minimum: 2.35
+        maximum: 2.65
+    terminal_length:
+        minimum: 0.35
+        maximum: 0.85
+    size_info: '(Body size source: IPC-SM-782 page 72, https://www.pcb-3d.com/wordpress/wp-content/uploads/ipc-sm-782a_amendment_1_and_2.pdf)'
+
+SMD_1812:
```

Comment:
Should this be R_1812 instead of SMD_1812?

Reply:
Yes. Initially that broke my branch but I think I know better how to fix it now so I went for it again. However, the branch can still be merged so I guess we're good.